.github/workflows/build.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
-        java: [8, 8.0.192, 11, 11.0.2, 13, 13.0.4, 15, 16-ea]
+        os: [macos-latest]
+        java: [8, 11, 13, 15, 16-ea]
```

Comment:
Java 16 is GA now IIRC, so I think this should be updated to remove the `-ea` part. Also, `actions/setup-java@v2` is [available now](https://github.com/actions/setup-java/blob/main/docs/switching-to-v2.md).

Reply:
Good shout :bow:  Updating.